sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Infrastructure/PerfTestEnvironment.cs

```diff
@@ -26,13 +26,13 @@ private PerfTestEnvironment()
         /// <summary>
         /// The endpoint of the Text Analytics account to test against.
         /// </summary>
-        /// <value>The endpoint, read from the "TEXT_ANALYTICS_ENDPOINT" environment variable.</value>
-        public string Endpoint => GetVariable("TEXT_ANALYTICS_ENDPOINT");
+        /// <value>The endpoint, read from the "TEXTANALYTICS_ENDPOINT" environment variable.</value>
+        public string Endpoint => GetVariable("TEXTANALYTICS_ENDPOINT");
```

Comment:
This means having to re-record all sessions - which you did - and git does not store diffs. It stores whole files. For just renaming variables, is growing the repo even further worth it? Maybe a TODO for next time you need to rerecord for a new API version or something?

Reply:
Sounds good. I undid the change.